packages/components/src/components/news-ticker/news-ticker-children.tsx

```diff
@@ -1,12 +1,18 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 
+type TNewsTickerChildren = {
+    animation_duration: number;
+    is_exceeding_parent?: boolean;
```

Comment:
` is_exceeding_parent `was a required param before. Any reason why we make it optional?

Reply:
In the component we do have a fallback if no value is provided. So making it mandatory has no benefits